continuum/task_set.py

```diff
@@ -19,24 +20,27 @@ class TaskSet(TorchDataset):
     """
 
     def __init__(
-        self,
-        x: np.ndarray,
-        y: np.ndarray,
-        t: np.ndarray,
-        trsf: transforms.Compose,
-        data_type: str = "image_array"
+            self,
+            x: np.ndarray,
+            y: np.ndarray,
+            t: np.ndarray,
+            trsf: transforms.Compose,
+            data_type: str = "image_array"
     ):
-        self.x, self.y, self.t = x, y, t
+
+        self._x, self._y, self._t = x, y, t
```

Comment:
What was the rational behind prefixing by `_` here?

Reply:
I think we discussed it before, it is because they are private attribute

Reply:
can I mark this conv as "resolved"?

Reply:
I will assume yes :)